packages/test/test-service-load/src/utils.ts

```diff
@@ -108,6 +110,12 @@ class FileLogger extends TelemetryLogger implements ITelemetryBufferedLogger {
 			event.message.includes("FaultInjectionNack")
 		) {
 			event.category = "generic";
+		} else if (
```

Comment:
can't we fix this at the emission point, rather than here, use something like the ThresholdCounter

Reply:
in general we only use this to suppress error create in this test, not product errors

Reply:
What happens is the summarizer client actually closes. We would need to maintain state across the document across multiple clients. The problem with that is that we'd either have to send a signal or op to achieve that level of synchronization. Or we would have to go and keep an eye on the quorum - which complicates the code, just to remove some errors. Usually what happens is the same parent summarizer client is re-elected. Once the feature has completely rolled out to production, you can look at the task on my plate [AB#4370](https://dev.azure.com/fluidframework/internal/_workitems/edit/4370), I'd be comfortable altering the summarizer refresh flow. For now, we need both as both are in production.